src/pages/using-spark/components/button.mdx

```diff
@@ -115,14 +76,17 @@ Represents the second-highest priority action, following the Primary Button. The
   titleAttr="Secondary Button Example"
 />
 
+- There can be more than one on a screen.
```

Comment:
page instead of screen. I would search on these changes where screen should be replaced with "page"